packages/rsx/src/lib.rs

```diff
@@ -244,8 +250,11 @@ impl<'a> DynamicContext<'a> {
             }
 
             BodyNode::Text(text) if text.is_static() => {
-                let text = text.source.as_ref().unwrap();
-                quote! { ::dioxus::core::TemplateNode::Text{ text: #text } }
+                if let Some(text) = text.to_static() {
```

Comment:
can we just unwrap here instead of the let some unreachable?

Reply:
done